kartothek/io/dask/bag.py

```diff
@@ -67,6 +68,11 @@ def read_dataset_as_metapartitions_bag(
     -------
     A dask.bag object containing the metapartions.
     """
+    if partition_size and n_partitions:
+        raise ValueError(
+            "Please only one of the argument `partition_size` or `n_partitions`"
```

Comment:
Should be "Please *provide* only one [...]"

Reply:
Done!